packages/@atjson/offset-annotations/src/utils/social-urls.ts

```diff
@@ -349,13 +349,34 @@ function normalizeTelegramUrl(url: IUrl) {
     },
   };
 }
+function isRedditURL(url: IUrl) {
+  return (
+    (url.host === "www.redditmedia.com" && url.pathname.startsWith("/r/")) ||
+    url.host === "www.reddit.com"
+  );
+}
 
-export function identify(
-  url: IUrl
-): {
+function normalizeRedditURL(url: IUrl) {
+  let ref_source = "embed";
+  let ref = "share";
+  let embed = true;
+  return {
+    attributes: {
+      url: `https://www.redditmedia.com${url.pathname}?ref_source=${ref_source}&amp;ref=${ref}&amp;embed=${embed}`,
```

Comment:
You also need to pass through the `showmedia` and `created` params if they are present. You might also want to pass through the `theme` and `showedits` as well.

For example, users are able to generate an embed code like this from Reddit:
```
<iframe id="reddit-embed" src="https://www.redditmedia.com/r/NatureIsFuckingLit/comments/r4uas9/sea_sheep_are_one_of_the_few_animals_that_use/?ref_source=embed&amp;ref=share&amp;embed=true&amp;showmedia=false&amp;theme=dark&amp;showedits=false&amp;created=2021-11-29T15%3A12%3A51.249Z" sandbox="allow-scripts allow-same-origin allow-popups" style="border: none;" height="152" width="640" scrolling="no"></iframe>
```
This should produce an Iframe annotation with the a `url` attribute that includes the above params.